Demos/MungoHealer iOS-Demo/MungoHealer iOS-Demo/ViewController.swift

```diff
@@ -13,16 +13,22 @@ struct MyLocalizedError: LocalizedError {
 }
 
 struct MyBaseError: BaseError {
+    var debugDescription: String?
```

Comment:
Why was this added? Is this really necessary for Swift 5 support?

Reply:
When `debugDescription` is not explicitly defined, it triggers the error:

`Type 'MyBaseError' does not conform to protocol 'BaseError'`

Reply:
This still doesn't clarify why adding the `debugDescription` too all types is the correct migration of the Swift 4.2 behavior to Swift 5. Which change in Swift did make this necessary? What is the recommended way to migrate from Swift 4.2 to Swift 5? These questions can only be answered if the rationale behind this change is clear. Therefore we should try to find the Swift Evolution prorposals which made this change to Swift. Only then it will be clear if and why this is the correct solution to the error.

Reply:
OK I've figure it out: what we need to do in this case, is just defining a default value with a proper access (e.g. `public`) at the protocol level. A default value with an appropriate access seems to be a must for an optional property.